homeassistant/components/triggercmd/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""The TRIGGERcmd component."""
+
+from __future__ import annotations
+
+from triggercmd import ha
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+PLATFORMS = [
+    Platform.SWITCH,
+]
+
+type TriggercmdConfigEntry = ConfigEntry[ha.Hub]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: TriggercmdConfigEntry) -> bool:
+    """Set up TRIGGERcmd from a config entry."""
+    entry.runtime_data = ha.Hub(entry.data["token"])
```

Comment:
Can we verify the service works here?

Reply:
Sorry, I don't understand yet.  Please explain how or what it should verify.  
Are you thinking it should call the triggercmd api here to test the token again?  That seems redundant because the validate_input function in config_flow.py is already doing that.